src/storyblok-auth-api/auth-handler.ts

```diff
@@ -17,11 +20,31 @@ export const authHandler = (
       res.writeHead(400).end()
       return
     }
+
+    const internalAdapter = createInternalAdapter({
+      req,
+      res,
+      adapter: cookieAdapter,
+    })
+
+    const query = getQueryParams(url)
+    const isInitRequest = query.get('init_oauth')
+
+    // NOTE: This is a workaround to remove a stale cookie in case the user has reinstalled the plugin.
+    if (isInitRequest === 'true') {
+      await internalAdapter.removeItem(sessionIdentifier(params))
+    }
```

Comment:
This logic removes stale cookie when reinstalling the app and is connected to this PR: [feat(nuxt-base): remove init_oauth stale cookie deletion and move to app-extension-auth](https://github.com/storyblok/space-tool-plugins/pull/66)